src/main/java/org/folio/circulation/domain/policy/Period.java

```diff
@@ -175,4 +190,27 @@ public int toMinutes() {
     }
   }
 
+  @Override
+  public boolean equals(Object other) {
+    if (this == other) {
+      return true;
+    }
+
+    if (other == null || getClass() != other.getClass()) {
+      return false;
+    }
+
+    Period period = (Period) other;
+    return timePeriod().equals(period.timePeriod());
+  }
+
+  @Override
+  public int hashCode() {
+    return timePeriod().hashCode();
+  }
+
+  @Override
+  public int compareTo(Period otherPeriod) {
+    return Integer.compare(toMinutes(), otherPeriod.toMinutes());
+  }
```

Comment:
Please could some unit tests be added to demonstrate the behaviour of equality and comparison

Reply:
Added.